editor/src/components/canvas/canvas.ts

```diff
@@ -287,11 +281,21 @@ const Canvas = {
             return true
           }
         default:
-          const _exhaustiveCheck: never = searchType
-          throw new Error(`Unknown search type ${JSON.stringify(searchType)}`)
+          assertNever(searchType)
       }
     })
   },
+  getMousePositionCanvasArea(canvasPosition: CanvasPoint | null): CanvasRectangle | null {
+    if (canvasPosition == null) {
+      return null
+    }
+    return canvasRectangle({
+      x: canvasPosition.x - 1,
+      y: canvasPosition.y - 1,
```

Comment:
isn't this shifting the click by a single pixel to the left? shouldn't this just be x;y here? or are these changes matching what was on master?

Reply:
this mimics what's on master without breaking the tests - the skew of 1 px to the northwest is compensated by the 1x1 area :)